gadal/gadal_types/generate_schema.py

```diff
@@ -1,15 +1,52 @@
 from .data_types import *
 import os
 
-all_classes = [StateArray, SwitchStates, CapacitorStates, RegulatorStates, CostArray, RealCostFunctions, ReactiveCostFunctions, RealWholesalePrices, ReactiveWholesalePrices, OperationalCosts, MeasurementArray, VoltagesMagnitude, VoltagesAngle, VoltagesReal, VoltagesImaginary, CurrentsMagnitude, CurrentsAngle, CurrentsReal, CurrentsImaginary, PowersMagnitude, PowersAngle, PowersReal, PowersImaginary, PowersImaginary, SolarIrradiances, Temperatures, WindSpeeds, StatesOfCharge, Topology, AdmittanceSparse, AdmittanceMatrix]
+all_classes = [
+    StateArray,
+    SwitchStates,
+    CapacitorStates,
+    RegulatorStates,
+    CostArray,
+    RealCostFunctions,
+    ReactiveCostFunctions,
+    RealWholesalePrices,
+    ReactiveWholesalePrices,
+    OperationalCosts,
+    MeasurementArray,
+    VoltagesMagnitude,
+    VoltagesAngle,
+    VoltagesReal,
+    VoltagesImaginary,
+    CurrentsMagnitude,
+    CurrentsAngle,
+    CurrentsReal,
+    CurrentsImaginary,
+    PowersMagnitude,
+    PowersAngle,
+    PowersReal,
+    PowersImaginary,
+    PowersImaginary,
+    SolarIrradiances,
+    Temperatures,
+    WindSpeeds,
+    StatesOfCharge,
+    Topology,
+    AdmittanceSparse,
+    AdmittanceMatrix,
+    Injection,
+    Command,
+    CommandList,
+    VVControl,
+    VWControl,
+    InverterControl,
+    InverterControlList,
+]
 
-all_data = ''
+if not os.path.exists("schemas"):
+    os.makedirs("schemas")
 
-if not os.path.exists('schemas'):
-    os.makedirs('schemas')
 for klass in all_classes:
+    print(klass)
```

Comment:
commentthe print statement. Add to logger if necessary

Reply:
Fixed.